json-guides/microprofile-config.json

```diff
@@ -11,18 +11,18 @@
             "title": "What you'll learn",
             "guide_duration" : "25 minutes",
             "description": [
-              "Explore how to inject configuration data into a microservice without repackaging the application each time the underlying runtime environment changes.  <a href='https://microprofile.io/' target='_blank'>MicroProfile</a> makes building configurable microservices easy with its <a href='https://github.com/eclipse/microprofile-config' target='_blank'>MicroProfile Config</a> feature. Application configuration properties from multiple sources are combined into a single set of configuration properties and accessed from a single API by your application.<br>",
+              "Explore how to inject configuration data into a microservice without repackaging the application each time the underlying runtime environment changes.  <a href='https://microprofile.io/' target='_blank'>MicroProfile</a> makes building configurable microservices easy with its <a href='https://github.com/eclipse/microprofile-config' target='_blank'>MicroProfile Config</a> feature. Your application combines  application configuration properties from multiple sources are combined into a single set and accesses the properties from a single API by your application.<br>",
```

Comment:
I think you have some extra words in this sentence "Your application combines application..."
Shouldn't it be, "Your application combines application configuration properties from multiple sources into a single set and accesses the properties from a single API."

Reply:
done